projector-plugin/src/main/kotlin/ProjectorService.kt

```diff
@@ -160,7 +155,30 @@ class ProjectorService : PersistentStateComponent<ProjectorConfig> {
   }
 
   companion object {
-    val instance: ProjectorService by lazy { ServiceManager.getService(ProjectorService::class.java)!! }
+    private val instance: ProjectorService by lazy { ServiceManager.getService(ProjectorService::class.java)!! }
+
+    fun enable() = instance.enable()
+    fun disable() = instance.disable()
+    fun activate() = instance.activate()
+
+    var enabled: EnabledState
+      get() = instance.enabled
+      set(value) {
+        instance.enabled = value
+      }
+
+    val host: String? get() = instance.config.host
+    val port: String? get() = instance.config.port
+
+    val isSessionRunning: Boolean get() = instance.currentSession != null
+    var currentSession: Session
+      get() {
+        require(isSessionRunning) { "Current sessino is not available - no active sessions" }
```

Comment:
```suggestion
        check(isSessionRunning) { "Current session is not available - no active sessions" }
```

Fix typo and change checker function to more appropriate